src/cms/views/events/event_view.py

```diff
@@ -173,6 +173,12 @@ def post(self, request, **kwargs):
             event_form.add_error_messages(request)
             event_translation_form.add_error_messages(request)
             recurrence_rule_form.add_error_messages(request)
+        elif (
+            event_translation_form.instance.status == status.AUTO_SAVE
+            and not event_form.has_changed()
+            and not event_translation_form.has_changed()
+        ):
```

Comment:
```suggestion
            and not recurrence_rule_form.has_changed()
        ):
```
I think the recurrence rule form has to be checked here as well...